Sources/NIO/CircularBuffer.swift

```diff
@@ -692,3 +692,16 @@ extension CircularBuffer {
         return self.verifyInvariants() && self.unreachableAreNil()
     }
 }
+
+extension CircularBuffer: Equatable where Element: Equatable {
+    public static func ==(lhs: CircularBuffer, rhs: CircularBuffer) -> Bool { lhs.count == rhs.count && zip(lhs, rhs).allSatisfy(==)
```

Comment:
No prizes for avoiding the extra line here: add a newline after `{` :wink:

Reply:
Oops, copy-pasted it from GitHub into another branch and the newline was lost 😅

Reply:
@Lukasa I don't understand what's happening, because when I go directly to my branch here: https://github.com/adtrevor/swift-nio/commit/25553e86b4b44fdc150207da1333841f5ce5108b
Everything is correct, and it's also the case in Xcode ! 😮